testing/types.go

```diff
@@ -49,6 +49,7 @@ type TestBody struct {
 // and the Test asserts truth of falsity of this statement.
 type JwtBody struct {
 	Subject string   `json:"sub" yaml:"sub"`
+	UserId  string   `json:"user_id" yaml:"user_id"`
```

Comment:
`Claims map[string]string `json:"claims,omitempty" yaml:"claims,omitempty"`

Reply:
I updated to include the changes proposed but this would require all tests to be updated from
```yaml
    - name: "test-case-01"
      expect: false
      token:
        sub: "adam@email.com"
        roles:
          - ADMIN
      resource:
        path: "/endpoint"
        method: POST
```
to
```yaml
    - name: "test-case-01"
      expect: false
      token:
        claims:
          sub: "adam@email.com"
          roles:
            - ADMIN
      resource:
        path: "/endpoint"
        method: POST
```
Would we want to instead change `JwtBody` to be `type JwtBody map[string]interface{}` to preserve the original structure?

Reply:
no, to support this use case you would simply do (in the test cases that you care about):
```
    - name: "test-case-01"
      expect: true
      token:
        sub: "adam@email.com"
        roles:
            - ADMIN
        claims: {
          user_id: "1234_567"
        }
      resource:
        path: "/test/users/1234_567"
        method: GET
```
something like that

Reply:
Gotcha, updated. Confirmed this worked as expected in an OPA project using this project to run its tests.